rabbitinahat/src/main/java/org/ohdsi/rabbitInAHat/DetailsPanel.java

```diff
@@ -287,8 +287,14 @@ public void showTable(Table table) {
 			this.table = table;
 			nameLabel.setText(table.getName());
 			DecimalFormat formatter = new DecimalFormat("#,###");

```

Comment:
How about using `NumberFormat.getInstance()` here for a locale-dependent formatting?